web/src/main/webapp/xslt/base-layout-cssjs-loader.xsl

```diff
@@ -281,6 +282,18 @@
       <script type="text/javascript" src="{$uiResourcesPath}lib/angular.ext/ui-ace.js?v={$buildNumber}"></script>
     </xsl:if>
 
+    <!-- Load variables that are required for user group profile management. -->
+    <xsl:if test="$angularApp = 'gn_admin'">
```

Comment:
Could you move this code above and change the following?

` <xsl:if test="$angularApp = 'gn_search' or $angularApp = 'gn_login'">`

to 

` <xsl:if test="$angularApp = 'gn_search' or $angularApp = 'gn_login' or $angularApp = 'gn_admin'">`

Seems like you would only need to add 

`gnGlobalSettings.isUserGroupUpdateEnabled = <xsl:value-of select="$isUserGroupUpdateEnabled"/>;`

since the others variables already exists there.

I think this would remove some duplication and make the code cleaner.

Reply:
moved. They are not totally identical. The  gn_search and gn_login has more keycloak logic but gn_admin has nothing.